azure-pipelines.yml

```diff
@@ -7,6 +7,7 @@ pr:
   branches:
     include:
     - master
+    - release-v1.6

```

Comment:
Maybe we should add `- release-*` to the main branch to get this on all future release branches